tests/cases/conformance/classes/members/privateNames/privateNameInTypeQuery.ts

```diff
@@ -0,0 +1,11 @@
+// @strict: true
+// @target: esnext
+
+class C {
+    #a = 'a';
+
+    constructor() {
+        const a: typeof this.#a = ''; // Ok
+        const b: typeof this.#a = 1;  // Error
+    }
+}
```

Comment:
```suggestion
class C {
    #a = 'a';

    constructor() {
        const a: typeof this.#a = ''; // Ok
        const b: typeof this.#a = 1;  // Error
    }
}

const instanceOfC = new C();
const a: typeof instanceOfC.#a = ''; // Error
```

Reply:
@DanielRosenwasser I've updated the test.